filters/auth/grantlogout.go

```diff
@@ -0,0 +1,192 @@
+//
+// grantLogout filter
+//
+// Used for revoking access/refresh tokens generated by the oauthGrant() filter.
+//
+
+package auth
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/filters"
+)
+
+const (
+	GrantLogoutName           = "grantLogout"
+	revokeTokenKey            = "token"
+	revokeTokenTypeKey        = "token_type_hint"
+	refreshTokenType          = "refresh_token"
+	accessTokenType           = "access_token"
+	unsupportedTokenTypeError = "unsupported_token_type"
```

Comment:
Linter rule that is not yet enabled, but we should one day enforce it, says:
unsupportedTokenTypeError -> errUnsupportedTokenType

Reply:
Renamed ✅